data/css/blog.scss

```diff
@@ -0,0 +1,111 @@
+@import "project-setup";
```

Comment:
Anything from this file we can put in the default theme? This contains the things you added to "make it pretty" but if there's anything we can add that makes the default theme prettier by default, then we should do that.